addons/mail/static/tests/new/activity/activityData.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module */
+
+export const activityData = {
+    activity_category: "default",
+    activity_decoration: false,
+    activity_type_id: [4, "To Do"],
+    automated: false,
+    can_write: true,
+    chaining_type: "suggest",
+    create_date: "2022-11-09 12:10:54",
+    create_uid: [2, "Mitchell Admin"],
+    date_deadline: "2022-11-14",
+    display_name: "To Do",
+    has_recommended_activities: false,
+    icon: "fa-tasks",
+    id: 8,
+    mail_template_ids: [],
+    note: false,
+    previous_activity_type_id: false,
+    recommended_activity_type_id: false,
+    request_partner_id: false,
+    res_id: 7,
+    res_model: "project.task",
+    res_model_id: [391, "Task"],
+    res_name: "Room 2: Decoration",
+    state: "planned",
+    summary: false,
+    user_id: [2, "Mitchell Admin"],
+    write_date: "2022-11-09 12:10:54",
+    write_uid: [2, "Mitchell Admin"],
+};
```

Comment:
Define a `addActivity()` method in `TestServer` instead, like we did with `addChannel`/`addMessage`/`addPartner`/etc.

This would incentivise to let each test define data (so not big data shared between tests, and hard to maintain), and keep data as small as necessarily desirable for proper testing.